relayer_test.go

```diff
@@ -0,0 +1,103 @@
+package sequence_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/0xsequence/ethkit/ethcoder"
+	"github.com/0xsequence/go-sequence"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetReceiptOfTransaction(t *testing.T) {
+	// Ensure dummy sequence wallet from seed 1 is deployed
+	wallet, err := testChain.DummySequenceWallet(1)
+	assert.NoError(t, err)
+	assert.NotNil(t, wallet)
+
+	isDeployed, err := wallet.IsDeployed()
+	assert.NoError(t, err)
+	assert.True(t, isDeployed)
+
+	// Create normal txn of: callmockContract.testCall(55, 0x112255)
+	callmockContract := testChain.UniDeploy(t, "WALLET_CALL_RECV_MOCK", 0)
+	calldata, err := callmockContract.Encode("testCall", big.NewInt(55), ethcoder.MustHexDecode("0x112255"))
+	assert.NoError(t, err)
+
+	nonce, err := wallet.GetNonce()
+	assert.NoError(t, err)
+
+	stx := &sequence.Transaction{
+		To:            callmockContract.Address,
+		Data:          calldata,
+		Value:         big.NewInt(0),
+		GasLimit:      big.NewInt(190000),
+		DelegateCall:  false,
+		RevertOnError: false,
+		Nonce:         nonce,
+	}
+
+	err = signAndSendRawTransaction(t, wallet, stx)
```

Comment:
seems we can prob move these methods along with other helpers directly on the *TestChain, we can do it later tho, but feels like the right thing to do.